src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -129,11 +129,26 @@ export const Finish = (props: OwnProps) => {
               InfluxDB.
             </p>
           </ResourceCard>
+          {props.wizardEventName === 'cliWizard' && (
+            <ResourceCard className="homepage-wizard-next-steps">
+              <SafeBlankLink
+                href="https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/"
+                onClick={() =>
+                  handleNextStepEvent(props.wizardEventName, 'cliCommands')
+                }
+              >
+                <h4>{BookIcon}CLI Commands</h4>
+              </SafeBlankLink>
+              <p>See the full list of CLI commands and how to use them.</p>
+            </ResourceCard>
+          )}
```

Comment:
I was just informed sample cards will be added to this finish page. If so, I wonder if we should temporarily have the cli commands card on a new row (under influx university and key concepts). Like this: 
<img width="1456" alt="Screen Shot 2022-07-27 at 11 48 45 AM" src="https://user-images.githubusercontent.com/66275100/181304183-ef918d2d-f4b9-4762-afb8-da4c10e2aaeb.png">